.run/All Eff Sats.run.xml

```diff
@@ -1,6 +1,6 @@
 <component name="ProjectRunConfigurationManager">
```

Comment:
.run folder should be ignored and not present in this PR, so none of the XML files should be coming in.

Reply:
Resolved.